### PR TITLE
Further markup corrections part 2

### PR DESCRIPTION
**Pull request by manuel489**
Again, necessary markup corrections in SI Brochure.

Actions taken:

* Added `cdot` operator to units symbols
* Few markup corrections.

### CODE REVIEW

File: sources/sections-a1-en/13-13th-cgpm.adoc [hunk 1]
```diff
@@ -123,8 +123,8 @@ The 13th Conférence Générale des Poids et Mesures (CGPM),
 |===
 | wave number | 1 per metre | stem:["unitsml(m^(-1))"]
 | entropy | joule per kelvin | stem:["unitsml(J//K)"]
-| specific heat capacity | joule per kilogram kelvin | stem:["unitsml(J//kg*K)")] (((heat capacity)))(((joule (stem:["unitsml(J)"]))))
-| thermal conductivity | watt per metre kelvin | stem:["unitsml(W//m*K)")]
+| specific heat capacity | joule per kilogram kelvin | stem:["unitsml(J)"//("unitsml(kg)"*"unitsml(K)")] (((heat capacity)))(((joule (stem:["unitsml(J)"]))))
```
Comment (opoudjis) [inline sources/sections-a1-en/13-13th-cgpm.adoc:126]: I don't understand why you are doing this; is it for rendering? "unitsml(J//kg*K)" is a single unit. "unitsml(J)"//("unitsml(kg)"*"unitsml(K)" is indeed an operation involving three units, but the semantic intent is of a single unit generated from those three units.

**Comment by opoudjis** [comment]
In general, I think operations between unitsml expressions don't make sense, and go against the purpose of UnitsML markup, which is to identify discrete units in the text. I will regenerate the brochure based on these changes, and I don't anticipate that BIPM will look at the semantic markup too carefully, but I don't think it's the right thing to do.

File: sources/sections-a1-en/13-13th-cgpm.adoc (hunk 1 above)
Comment (manuel489) [reply sources/sections-a1-en/13-13th-cgpm.adoc:126]: I just wanted to match with the original format, which I had understood, was the desiderata.
But yes, I agree with your argument. Is there a way to include the `cdot` in the units?

**Comment by opoudjis** [comment]
I am merging, but `unitsml(expression, multiplier: &#xb7;)` is the way to do these properly.

**Pull request merged**